client/src/modules/Schedule/components/TableView/TableView.tsx

```diff
@@ -47,7 +51,8 @@ const getColumns = ({
           </a>
         );
       },
-      ...getColumnSearchProps('name'),
+      filteredValue: filteredInfo.name || null,
```

Comment:
As I can see, `filteredInfo.name` if type of `FilterValue | null`, what's the purpose of `|| null` here?

Reply:
I think, it left since some of my "experiments" with this field. Fixed, thank you